src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Bit.Core.Settings;
+using LaunchDarkly.Sdk.Server;
+using LaunchDarkly.Sdk.Server.Integrations;
+
+namespace Bit.Core.Services;
+
+public class LaunchDarklyFeatureService : IFeatureService, IDisposable
+{
+    private readonly LdClient _client;
+
+    public LaunchDarklyFeatureService(
+        IGlobalSettings globalSettings)
+    {
+        var ldConfig = Configuration.Builder(globalSettings.LaunchDarkly?.SdkKey);
+
+        if (string.IsNullOrEmpty(globalSettings.LaunchDarkly?.SdkKey))
```

Comment:
ℹ️ Interesting find from this latest revision and why I removed some tests -- this implementation will return the `IsOnline` property as `false` when this file can't be loaded *independent of* the offline flag below, so there ends up being a lot of offline states.

The LaunchDarkly SDK will spit out some errors about an invalid SDK key when it also doesn't have a file fallback, so I avoid that with more active offline sets.

Reply:
> when this file can't be loaded

which file; the flag path?